clients/drcachesim/tests/warmup-zeros.templatex

```diff
@@ -0,0 +1,27 @@
+Hello, world!
+---- <application exited with code 0> ----
+Cache simulation results:
+Core #0 \(1 thread\(s\)\)
+  L1I stats:
+    Warmup hits:                  *[0-9,\.]*..
```

Comment:
These should only accept 0, not any digit sequence, right?

Reply:
Yes, I missed the regex update for L1I and L1D. Fixed in latest commit.